Formula/fluent-bit.rb

```diff
@@ -1,8 +1,8 @@
 class FluentBit < Formula
-  desc "Data Collector for IoT"
+  desc "Fast and Lightweight Logs and Metrics processor for Linux, BSD, OSX and Windows"
```

Comment:
In our case it's only macOS and Linux anyway.
```suggestion
  desc "Fast and Lightweight Logs and Metrics processor"
```